graphenebase/betting/market.py

```diff
@@ -0,0 +1,99 @@
+import re
+import json
+
+from graphenebase.types import Id, Uint16, OrderedDict
+from graphenebase.objects import GrapheneObject
+
+
+markets = [
```

Comment:
It is better practice to name such constants (e.g. variables that won't be changed during process run) uppercase. Other words I suggest to rename it to `MARKETS`